keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/IntegerVertex.java

```diff
@@ -39,6 +40,18 @@ public IntegerVertex(long[] shape) {
         super(shape);
     }
 
+    @Override
+    public IntegerVertex print() {
```

Comment:
This has the same functionality as the print on the `Vertex` class so I think you don't need to override. See my comment on `Vertex.java` for something that should work.